src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -1,5 +1,325 @@
+    "transports": ["http"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/resourcemanager/resourcemanager.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Cloud Resource Manager API",
+      "compute":"java",
+      "site":"https://cloud.google.com/speech/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/resourcemanager/apidocs/com/google/cloud/compute/package-summary.html",
+      "infotip":"The com.google.cloud.speech.resourcemanager packages",
+      "launchStage" : "alpha",  
+      "packages" : ["com.google.cloud.resourcemanager"],
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-resourcemanager",
+      "languageLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-resourcemanager",
+         "version" : "0.23.0-alpha"
+       }
+      }
+    ]
+  },  {
+    "name": "Data Loss Prevention",
+    "documentation": "https://cloud.google.com/dlp/docs/reference/libraries",
+    "description":"The DLP API enables clients to detect the presence of Personally Identifiable Information (PII) and other privacy-sensitive data in user-supplied, unstructured data streams such as text blocks and images.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/dlp.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Data Loss Prevention API",
+      "compute":"java",
+      "site":"https://cloud.google.com/dlp/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/dlp/v2beta1/apidocs/com/google/cloud/compute/package-summary.html",
+      "infotip":"The com.google.cloud.dlp.v2beta1 packages",
+      "launchStage" : "alpha",  
+      "packages" : ["com.google.cloud.dlp.v2beta1"],
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-dlp",
+      "languageLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-dlp",
+         "version" : "0.23.0-alpha"
+       }
+      }
+    ]
+  },
+  {
+    "name": "Video Intelligence",
+    "documentation": "https://cloud.google.com/video-intelligence/docs/",
+    "description":"The Google Cloud Video Intelligence API enables developers to annotate videos stored in Google Cloud Storage with video and frame-level (1 fps) contextual information. The API supports common video formats including .mov, .mp4, and .avi.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/video-intelligence.svg",
```

Comment:
404.  We sure could use a consistent placement of logos!  This URL works: https://cloud.google.com/images/products/video-intelligence/video-intelligence-icon.svg  (link snarfed from https://cloud.google.com/products/machine-learning/)